ntp.go

```diff
@@ -24,8 +24,12 @@ var (
 	timeSynced            = false
 	timeSyncRetryInterval = 0 * time.Second
 	defaultNTPServers     = []string{
+		"pool.ntp.org",
 		"time.cloudflare.com",
+		"time.google.com",
```

Comment:
I would avoid adding time.google.com mixed with these other sources. This time source does leap second smear, which will not play well with non-smear time sources.

Reply:
That's a good point, I doubt we'll ever get to there since presumably poot.ntp.org will always return (and google is way down...) but happen to remove it. I think there are _also_ get smeared returns from others available, but in this application, we're really not concerned with leap-second accuracy ;)

Reply:
Who should make that canonical call? I'm happy to adjust if someone tells me what to use... and mine is on order, so hopefully it'll show soon :)